spec/converterbase_spec.rb

```diff
@@ -293,6 +293,105 @@ def to_ruby(match, m1, m2, open_close)
                "追撃をしない俺に振り返り、｜再度《パワースラッシュ 》"
       end
     end
+
+    describe "｜を使ったルビ位置の指定" do
+      it "《 》基本" do
+        expect(to_ruby( \
+               "｜山田太郎《やまだたろう》",
```

Comment:
__[rubocop]__
* Layout/FirstParameterIndentation: Indent the first parameter one step more than `to_ruby( \`.

<!-- Comment by [SideCI](https://sideci.com) -->